### ISSUE TITLE
RawJSON doesn't trim newline

### ISSUE THREAD

**Issue by byrnedo** [rs/zerolog#438]
If you're adding a json response that was written using `json.NewEncoder(w).Encode(..)`, there's a trailing newline (by design).
The newline is then left un-escaped in the log entry if you use `RawJSON` to attach it.
Could zerolog trim the content from newlines for us?

**Comment by rs** [comment]
RawJSON does not make any check, the caller is responsible for sending the right input.

**Comment by byrnedo** [comment]
The thing is, isn't that technically a valid json response?

**Comment by rs** [comment]
Not in this context. Basically this method gives you the freedom to add some raw bytes. You are responsible to make the final output valid.

**Comment by mitar** [comment]
Yes, this is the issue of `Encode` adding newlines which is different from `json.Marshal`.
